### PR TITLE
Close sidebar when clicked outside sidebar

### PR DESCRIPTION
**Pull request by Solai-Adithya**
Proposed fix for issue #60 , Now the sidebar collapses when either the user clicks outside the sidebar or clicks the collapse icon

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -106,19 +106,37 @@ function writeEmoji(emoji) {
...
+    document.body.addEventListener('click', function(event) {
+      var sidebar = document.getElementsByClassName('left')[0];
+      if(!sidebar.contains(event.target)) {  //In case the user clicks outside the sidebar, the sidebar closes
+        CloseSidebar(); 
+      }
+    }, false); 
   }
-}
\ No newline at end of file
+
```
Comment (LalitNM) [inline public/script.js]: Delete this line.

File: public/script.js [hunk 2]
```diff
@@ -106,19 +106,37 @@ function writeEmoji(emoji) {
 
 // For sidebar to work on small screens
 
+function CloseSidebar() {
```
Comment (LalitNM) [inline public/script.js]: @Solai-Adithya everything else looks good but change the first letter of all function names to lowercase, this is the convention we have followed in this repository.

**Review by LalitNM — CHANGES_REQUESTED** [review]

File: public/script.js (hunk 2 above)
Comment (Solai-Adithya) [reply public/script.js]: Yup! Done

File: public/script.js (hunk 1 above)
Comment (Solai-Adithya) [reply public/script.js]: Done!

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
Merged :rocket: Good work @Solai-Adithya!

**Pull request merged**